src/Calypso-Browser/ClyNoChildren.class.st

```diff
@@ -0,0 +1,32 @@
+"
+I am ClyNoChildren, an exception indicating that an operation on a parent in a parent-child structure failed due to the absence of child elements. 
+"
+Class {
+	#name : #ClyNoChildren,
```

Comment:
Please add a suffix Error to the name (ClyNoChildrenError). But with  #supportsExpansion logic you will not need this change.

Reply:
Done